app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -126,6 +123,13 @@
     private ImageView arcPointer;
     private LinearLayout infoLayout;
 
+    private LinearLayout touchView;
+    private WindowManager.LayoutParams touchViewParams;
+    private Handler screenScanHandler;
+    private Runnable screenScanRunnable;
```

Comment:
The code would be easier to navigate if this were final or we used a non-anonymous subclass.